src/onSuccess.js

```diff
@@ -18,10 +18,10 @@ module.exports = async ({ utils, inputs, constants }) => {
 
   const isLocal = constants.IS_LOCAL
   const siteName = process.env.SITE_NAME
-  const deployPrimeUrl = process.env.DEPLOY_PRIME_URL
+  const deployUrl = process.env[onSuccessInputs.deployUrlEnvVar] || process.env.DEPLOY_PRIME_URL;
```

Comment:
I don't think we should default to `DEPLOY_PRIME_URL` if the requested env can't be found.

```suggestion
  const deployUrl = onSuccessInputs.deployUrlEnvVar ? process.env[onSuccessInputs.deployUrlEnvVar] : process.env.DEPLOY_PRIME_URL
```